### ISSUE TITLE
Hiring developers for exchange

### ISSUE THREAD

**Issue by cryptokq** [peatio/peatio#846]
Hiring a front end developer with 2+ years react experience and a back end developer comfortable with the peatio backend. 
Jobs are remote.
Please email me your github, resume and portfolio.

cryptokq@outlook.com
skype @cryptokq

**Comment by Apollo-IT** [comment]
Hi, I'm implementing new UI with React.js.
Also, I've added other main functions.
If you have interests, message me.
Regards.

**Comment by not-a-dev0** [comment]
@Apollo-IT Do you have a demo?

**Comment by Apollo-IT** [comment]
yes, I have

**Comment by not-a-dev0** [comment]
@Apollo-IT Can you share it?

**Comment by Apollo-IT** [comment]
No, I can't. Because it is not published yet and I've signed to NDA

**Comment by not-a-dev0** [comment]
Lol then you have no demo. @Apollo-IT
